src/main/scala/com/gu/mobile/notifications/football/lib/GoalNotificationBuilder.scala

```diff
@@ -88,8 +33,31 @@ object GenericPayload {
       s"${metadata.awayTeamScore} ${metadata.awayTeam.name}\n" +
       s"${goal.scorerName} ${goal.minute}min$extraInfo"
 
+    val  topics = Set(
+      Topic(
+        TopicTypes.FootballTeam,
+        metadata.homeTeam.id.toString
+      ),
+      Topic(
+        TopicTypes.FootballTeam,
+        metadata.awayTeam.id.toString
+      ),
+      Topic(
+        TopicTypes.FootballMatch,
+        metadata.matchID
+      ),
+      /** Yes, the old apps registered by the team NAME. FFS **/
```

Comment:
I think it means for fucks sake, This is code I moved from notifications-football to here.
I can remove the comment, or at least the ffs part